specification/paths/ShopEstimates.json

```diff
@@ -0,0 +1,64 @@
+{
+  "get": {
+    "tags": [
+      "ShopEstimates"
+    ],
+    "summary": "Get all available shop estimates for your account.",
+    "description": "This endpoint retrieves available shop estimates for your account.",
```

Comment:
is het niet: `available estimates for your shop` of geven we alle estimates voor al je shops?
Moet het endpoint trouwens niet `/shops/:id/estimates` worden?

Reply:
Ja goed punt maar dan maakt de relation shop ook niet meer uit want die heb je al aan gegeven in de url

Reply:
`shipment-estimates` for your shop 😃